KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift

```diff
@@ -19,6 +19,9 @@ final class KarhooBookingViewController: UIViewController, BookingView {
     private var bottomNotificationView: KarhooNotificationView!
     private var bottomNotificationViewBottomConstraint: NSLayoutConstraint!
     private var mapView: MapView = KarhooMKMapView()
+    private var bottomContainer: UIView!
+    private var asapButton: MainActionButton!
+    private var laterButton: MainActionButton!
```

Comment:
For consistency in terminology, what do you think of going with prebook or schedule instead of later?

Reply:
updated :)